setup.py

```diff
@@ -118,3 +118,8 @@
       entry_points=entry_points,
       **package_info
 )
+
+#Compiling the resource file for the gui
+import subprocess
```

Comment:
Doesn't seem to work on anaconda
```python
Using /Users/wkerzend/anaconda3/envs/tardis-devel/lib/python2.7/site-packages
Finished processing dependencies for tardis-sn==1.0.1
Traceback (most recent call last):
  File "setup.py", line 125, in <module>
    '-o', 'tardis/resource_rc.py'])
  File "/Users/wkerzend/anaconda3/envs/tardis-devel/lib/python2.7/subprocess.py", line 522, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/Users/wkerzend/anaconda3/envs/tardis-devel/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/Users/wkerzend/anaconda3/envs/tardis-devel/lib/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory```

Have a look here how they implement the compilation of `.qrc` files